boto3/data/health/2016-08-04/resources-1.json

```diff
@@ -0,0 +1,211 @@
+{
+  "service": {
+    "actions": {
+      "DescribeAffectedEntities": {
+        "request": { "operation": "DescribeAffectedEntities" },
+        "resource": {
+          "type": "AffectedEntity",
+          "identifiers": [
+            { "target": "eventArn", "source": "response", "path": "entities[].eventArn" },
+            { "target": "arn", "source": "response", "path": "entities[].entityArn" }
+          ],
+          "path": "entities"
+        }
+      },
+      "DescribeEventAggregates": {
+        "request": {
+          "operation": "DescribeEventAggregates",
+          "params": [
+            { "target": "aggregateField", "source": "input" }
+          ]
+        },
+        "resource": {
+          "type": "EventAggregate",
+          "identifiers": [
+            { "target": "value", "source": "response", "path": "eventAggregates[].aggregateValue" }
+          ],
+          "path": "eventAggregates[]"
+        }
+      }
+    },
+    "has": {
+      "Event": {
+        "resource": {
+          "type": "Event",
+          "identifiers": [
+            {
+              "target": "arn",
+              "source": "input"
+            }
+          ]
+        }
+      },
+      "EventType": {
+        "resource": {
+          "type": "EventType",
+          "identifiers": [
+            {
+              "target": "code",
+              "source": "input"
+            }
+          ]
+        }
+      },
+      "AffectedEntity": {
+        "resource": {
+          "type": "AffectedEntity",
+          "identifiers": [
+            {
+              "target": "eventArn",
+              "source": "input"
+            },
+            {
+              "target": "arn",
+              "source": "input"
+            }
+          ]
+        }
+      }
+    },
+    "hasMany": {
+      "Events": {
+        "request": {
+          "operation": "DescribeEvents"
+        },
+        "resource": {
+          "type": "Event",
+          "identifiers": [
+            { "target": "arn", "source": "response", "path": "events[].arn" }
+          ],
+          "path": "events[]"
+        }
+      },
+      "EventTypes": {
+        "request": {
+          "operation": "DescribeEventTypes"
+        },
+        "resource": {
+          "type": "EventType",
+          "identifiers": [
+            { "target": "code", "source": "response", "path": "eventTypes[].code" }
+          ],
+          "path": "eventTypes[]"
+        }
+      }
+    }
+  },
+  "resources": {
+    "AffectedEntity": {
+      "identifiers": [
+        { "name": "eventArn",
```

Comment:
Wasn't sure what name to give this to make it clear it was the eventARN vs the EntityARN.